tools/expotools/src/utils/getPackageViewFromRegistryAsync.ts

```diff
@@ -0,0 +1,37 @@
+import spawnAsync from '@expo/spawn-async';
+
+import { getExpoRepositoryRootDir } from '../Directories';
+
+type PlainObjectType<V = any> = {
+  [key: string]: V;
+}
+
+type PackageViewType = PlainObjectType & {
+  'name': string;
+  'dist-tags': PlainObjectType<string>;
```

Comment:
My opinion would be that by extracting a generic `PlainObjectType` we're losing an opportunity to make the keys understandable just by the type:
```ts
'dist-tags': { [key: string]: string };
// vs
'dist-tags': { [tag: string]: string };
```

I'm also curious if linter doesn't complain about stringified type keys